pumpAlerts.py

```diff
@@ -34,10 +35,9 @@ def sendMessage(message, isTPDA=False):
             bot.send_message(chat_id=chatId, text=message)
             break
         except:
-            print(
-                "Retrying to send tele message in",
+            logger.error(
+                "Retrying to send tele message in %ss.",
                 config["telegramRetryInterval"],
```

Comment:
Is the additional comma at the end of config["telegramRetryInterval"] intended?

Reply:
not inteded. will be fixt with next commit.